README.md

```diff
@@ -135,6 +140,7 @@ This can be changed by following config
   conn1 = Phoenix.ConnTest.build_conn()
   conn2 = conn1 |> Plug.Conn.assign(:scopes, ["scope1", "scope2"])
   conn3 = conn1 |> Plug.Conn.assign(:scopes, ["root_scope"])
+  conn4 = conn1 |> Plug.Conn.assign(:scopes, ["scope1"])
```

Comment:
can we  change order to 
```elixir
conn1 = Phoenix.ConnTest.build_conn()
conn2 = conn1 |> Plug.Conn.assign(:scopes, ["scope1"])
conn3 = conn1 |> Plug.Conn.assign(:scopes, ["scope1", "scope2"])
conn4 = conn1 |> Plug.Conn.assign(:scopes, ["root_scope"])
```
so we have more permissions with each conn ?

Reply:
I also changed order of actions to make table look like triangular matrix.